src/reducers/reducers.js

```diff
@@ -524,43 +524,108 @@ export function newlyAddedRecipe(state = initialNewlyAddedRecipe, action) {
 
 //EDIT RECIPE
 
-export function editRecipeReducer(state = initialBody, action) {
+const initialEditBody = {
```

Comment:
Has this been tested? Because I'm not sure about the reducer, here... :/

Should there be one big one, or no? (Cc: others!)

Reply:
i see it!!! definitely a mistake

Reply:
okay!! i don't see it